File: 0088-merge-sorted-array/0088-merge-sorted-array.js
/**
 * @param {number[]} nums1
 * @param {number} m
 * @param {number[]} nums2
 * @param {number} n
 * @return {void} Do not return anything, modify nums1 in-place instead.
 */
function merge(nums1, m, nums2, n) {
    let arr = [];


    for(let i = 0; i < m; i++){
        arr.push(nums1[i])
    }
    for(let i = 0; i < n; i++){
        arr.push(nums2[i])
    }

    arr.sort((a,b) => a - b);

    for (let i = 0; i < arr.length; i++) {
        nums1[i] = arr[i];
    }
}; 
 

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        res = []

        for i in range(m):
            res.append(nums1[i])
        for j in range(n):
            res.append(nums2[j])
        res.sort()
        for k in range(len(res)):
            nums1[k] = res[k]
